.github/config.yml

```diff
@@ -0,0 +1,33 @@
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!🚀⚡
+  
+  
+  Thank you and congrats🎉 for opening your very first issue in this project. We at hackStation provide a platform to share your work in number.
+  Please adhere to our [Code of Conduct](https://github.com/ashishnagpal2498/hackstation/blob/master/CODE_OF_CONDUCT.md).🙌
+  
```

Comment:
Can you please add -> Contribute md link also here. Check {How to contribute (link) ] to this project